explainer.md

```diff
@@ -463,48 +349,34 @@ For example:
   The screen reader sends the browser an increment *action* on the slider element
   in the accessibility tree.
 
-Currently, browsers implement partial support by accessible actions either by
-implementing built-in support for native HTML elements (for example, a native
-HTML `<input type="range">` already supports increment and decrement actions,
-and text boxes already support actions to set the value or insert text.
+Currently, browsers implement partial support for accessible actions 
+either by implementing built-in support for native HTML elements 
+(for example, a native HTML `<input type="range">` already supports 
+increment and decrement actions,
+and text boxes already support actions to set the value or insert text).
 
 However, there is no way for web authors to listen to accessible actions on
 custom elements.  For example, the custom slider above with a role of `slider`
 prompts a suggestion on VoiceOver for iOS to perform swipe gestures to increment
 or decrement, but there is no way to handle that semantic event via any web API.
 
-Accessible Actions gives web developers a mechanism to listen for accessible
-actions directly, by adding event listeners on an `AccessibleNode`.
-This is analogous to listening for user interaction events on a DOM node,
-except that the interaction event arrives via an assistive technology API,
-so it is directed to the accessible node first.
+The new UI Events would give web developers a mechanism 
+to listen for accessible actions directly, 
+by adding event listeners on an `Element`.
 
 For example, to implement a custom slider, the author could simply listen for
 `increment` and `decrement` events.
```

Comment:
`accessibleincrement` and `accessibledecrement`

Reply:
Done